app/models/api/Attachments.scala

```diff
@@ -23,7 +23,8 @@ import play.api.libs.json._
 case class Attachments(method: Option[AttachmentMethod] = None,
                        supplyVat1614a: Option[Boolean] = None,
                        supplyVat1614h: Option[Boolean] = None,
-                       supplySupportingDocuments: Option[Boolean] = None)
+                       supplySupportingDocuments: Option[Boolean] = None,
+                       additionalPartnersDocuments: Option[Boolean] = None)
```

Comment:
should we make this flag's existence mandatory for making the additional partner tasklist row complete, from the code it seems like all happypaths will define the value.